scipy/stats/stats.py

```diff
@@ -2306,6 +2311,186 @@ def zmap(scores, compare, axis=0, ddof=0):
         return (scores - mns) / sstd
 
 
+def _iqr_scale_to_num(scale):
+    """
+    A helper function to convert the 'scale' argument of `iqr` to a number.
+
+    scale: str
+        If scale is one of ['raw', 'normal'], this function will map the string
+        to a value.
+
+    The concept for this function is borrowed heavily from the `numpy.histogram`
+    helper function `_hist_optim_numbins_estimator`.
+    """
+    assert isinstance(scale, string_types)
+    # private function should not be called otherwise
+
+    # See https://en.wikipedia.org/wiki/Robust_measures_of_scale
+    scale_conversions = {'raw': 1.0,
+                         'normal': special.erfinv(0.5) * 2.0 * math.sqrt(2.0)}
+    try:
+        return scale_conversions[scale.lower()]
+    except KeyError:
+        raise ValueError("{0} not a valid scale for `iqr`".format(scale))
+
+
+def iqr(x, axis=None, range=(25, 75), scale='raw', nan_policy='propagate',
```

Comment:
Do you prefer `rng` or `interval`? I prefer the former because it's shorter and preserves the meaning of the 'R' in IQR. I only suggest the latter because it is a real word rather than an abbreviation, which seems more consistent with the other keywords.